Library/Formula/mpv.rb

```diff
@@ -0,0 +1,72 @@
+class Mpv < Formula
+  desc "Video player based on MPlayer/mplayer2"
+  homepage "https://mpv.io"
+  url "https://github.com/mpv-player/mpv/archive/v0.13.0.tar.gz"
+  sha256 "1372704fd8f5701ef7d60f347fd15fe536e23ef148279bf4a4415e72896912db"
+  head "https://github.com/mpv-player/mpv.git"
+
+  option "with-libmpv",          "Build shared libray."
+  option "without-docs",         "Disable building man page."
+  option "without-bundle",       "Disable compilation of a Mac OS X Application bundle."
+  option "without-zsh-comp",     "Install without zsh completion"
+
+  depends_on "pkg-config" => :build
+
+  depends_on "libass"
+  depends_on "ffmpeg"
+
+  depends_on "mpg123"      => :recommended
+  depends_on "jpeg"        => :recommended
+  depends_on "little-cms2" => :recommended
+  depends_on "lua"         => :recommended
+  depends_on "youtube-dl"  => :recommended
+
+  depends_on "libcaca"     => :optional
+  depends_on "libdvdread"  => :optional
+  depends_on "libdvdnav"   => :optional
+  depends_on "libbluray"   => :optional
+  depends_on "libaacs"     => :optional
```

Comment:
Don't bother doing all the alignment here and above; we don't really do this with our in-house style.